osmaxx-py/osmaxx/excerptexport/rest_api/views.py

```diff
@@ -35,3 +37,14 @@ def get_queryset(self):
 
 class BoundingGeometryFromExcerptDetail(BoundingGeometryFromExcerptMixin, generics.RetrieveAPIView):
     pass
+
+
+def country_geojson(request, pk):
+    country_id = pk
+    client = get_authenticated_api_client()
+    country_json_from_conversion_service = client.get_country_geojson(country_id)
```

Comment:
Passing `country_id` as keyword argument to `get_country_geojson(...)` you could inline the local `country_id` variable without hampering readability.